### ISSUE TITLE
Not up to current AQA Pseudocode Guidelines

### ISSUE THREAD

**Issue by davwheat** [gbaman/Python-To-AQA-Pseudocode#7]
The current AQA Pseudocode Teaching Guide is available here: https://filestore.aqa.org.uk/resources/computing/AQA-8520-TG-PC.PDF

As you can see `FUNCTION` is no longer used, instead being replaced by `SUBROUTINE`.

There are likely more changes but this is the one I noticed.

This repo doesn't seem to get updated any more, either.

I'm planning on making a new, updated version of this in C#.NET as a Windows executable.

**Comment by gbaman** [comment]
@davwheat Correct, I don't really maintain this repo anymore. The code itself was a very quick cobble together, that looking back on it now, if I built it again today, I would do it extremely different :)
(now I actually know a decent bit more Python)

It actually contains quite a few edge case bugs as well. But is available for anyone to fork and tinker with!

**Comment by davwheat** [comment]
Wow, that was quick.

I think I might fork and edit this as it would be a nice way to hone my skills during Year 10.

Thanks for the code. Time to get cracking!

**Comment by davwheat** [comment]
Created pull request #8 for this issue as well as a few other improvements/changes
